README.md

```diff
@@ -15,7 +15,7 @@ java -jar jfixtures-cmd-1.XX.jar <options>
 
 where options are:
 
-  -type [required] 
+  -type [optional]
```

Comment:
I would rather add a new `xml` item into `type`.
I know that is not consistent for us, from the code perspective. But should be much easier for user so they don't have to keep in mind one more flag.